lib/landable/traffic/user_tracker.rb

```diff
@@ -27,12 +27,12 @@ def record_page_view
           p.path         = request.path
           p.query_string = untracked_parameters.to_query
           p.request_id   = request.uuid
-
           p.click_id     = tracking_parameters['click_id']
-
           p.http_status  = response.status
 
-          p.visit_id     = @visit_id
+          # if instance var is lost, load session. If session is lost, make a new visit b/c there's no other way to retrieve.
+          p.visit_id     = @visit_id || (load and @visit_id) || (@visit_id = record_visit.visit_id)
```

Comment:
Line is too long. [99/80]<br>Use `&&` instead of `and`.<br>Unnecessary spacing detected.